internal/cloud/cloud.go

```diff
@@ -200,7 +206,7 @@ func syncFiles(username, hostname, configDir string) error {
 	time.Sleep(1 * time.Second)
 
 	// In a background routine, update the sync status in the cloud VM
-	go updateSyncStatus(mutagenSessionName, username, hostname, projectName)
+	go updateSyncStatus(mutagenSessionName, username, hostname, filepath.Base(projectPath))
```

Comment:
Should this be `hyphenatePath(projectPath)`? Would this status also have conflict?

Reply:
oh yeah! good call. Updated.